src/embedders/classification/contextual.py

```diff
@@ -34,6 +34,25 @@ class HuggingFaceSentenceEmbedder(TransformerSentenceEmbedder):
     def __init__(self, config_string: str, batch_size: int = 128):
         super().__init__(config_string, batch_size)
 
+    @staticmethod
+    def load(embedder: dict) -> "HuggingFaceSentenceEmbedder":
+        return HuggingFaceSentenceEmbedder(
+            config_string=request_util.get_model_path(embedder["config_string"]),
+            batch_size=embedder["batch_size"],
+        )
+
+    def to_json(self) -> dict:
+        return {
+            "cls": "HuggingFaceSentenceEmbedder",
+            "config_string": self.model.model_card_data.base_model,
```

Comment:
i think you mentioned that we can do this via a db query (or set a class variable based on a db query) or something since model_card_data doesn't have to exist. might have misunderstood something though